src/lib/ApxData/Sharing/Share.cpp

```diff
@@ -61,8 +60,12 @@ Share::Share(Fact *parent, QString type, QString dataTitle, QDir defaultDir, Fla
     connect(App::instance(), &App::loadingFinished, this, [this]() {
         QTimer::singleShot(1000, this, &Share::syncTemplates);
     });
-}
 
+    AppDirs::copyPath(AppDirs::res().absoluteFilePath("templates/share"),
+                      AppDirs::configs().absoluteFilePath(""),
+                      "vehicle",
+                      AppDirs::configBackups().absoluteFilePath(""));
+}
```

Comment:
`Share` is a base class for other classes in different modules ([Telemetry](https://github.com/uavos/apx-gcs/blob/51539cceb70539a95af60ea27132aa235601ceef/src/lib/ApxGcs/Telemetry/TelemetryShare.h#L37), [Mission](https://github.com/uavos/apx-gcs/blob/51539cceb70539a95af60ea27132aa235601ceef/src/lib/ApxGcs/Mission/MissionShare.h#L30), etc) thus it's going to copy many times the same files on every ancessor object invocation.

I'd suggest to copy these telemetry samples in this constructor: https://github.com/uavos/apx-gcs/blob/51539cceb70539a95af60ea27132aa235601ceef/src/lib/ApxGcs/Telemetry/Telemetry.cpp#L35